timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -446,6 +447,10 @@ private void registerCorruptionHandlerWrappedService(
     @VisibleForTesting
     static void verifyIsNewServiceInvariant(TimeLockInstallConfiguration install, ClusterConfiguration cluster) {
         if (!install.paxos().ignoreNewServiceCheck()) {
+            log.info(
+                    "Verifying the new service invariant",
```

Comment:
Either this is safe, or we've always got the safety wrong: https://github.com/palantir/atlasdb/blob/develop/timelock-agent/src/main/java/com/palantir/timelock/config/ClusterConfiguration.java#L61 (I believe the former, fwiw)